itests/hive-unit/src/test/java/org/apache/hive/jdbc/TestJdbcGenericUDTFGetSplits.java

```diff
@@ -38,9 +38,13 @@
 public class TestJdbcGenericUDTFGetSplits extends AbstractTestJdbcGenericUDTFGetSplits {
 
   @Test(timeout = 200000)
-  @Ignore("HIVE-23394")
   public void testGenericUDTFOrderBySplitCount1() throws Exception {
-    super.testGenericUDTFOrderBySplitCount1("get_splits", new int[]{10, 1, 0, 2, 2, 2, 1, 10});
+    super.testGenericUDTFOrderBySplitCount1("get_splits", new int[] { 10, 5, 0 });
```

Comment:
Why some splits removed here? If the dataset is not the same, won't hide the actual issue?

Reply:
+1